rpc/v6/pending_data_wrapper.go

```diff
@@ -0,0 +1,87 @@
+package rpcv6
+
+import (
+	"time"
+
+	"github.com/NethermindEth/juno/core"
+	"github.com/NethermindEth/juno/core/felt"
+	"github.com/NethermindEth/juno/sync"
+)
+
+func (h *Handler) PendingData() (*core.PendingData, error) {
+	pending, err := h.syncReader.PendingData()
+	if err != nil {
+		return nil, err
+	}
+	// If pending network is polling pending block and running on < 0.14.0
+	if pending.Variant() == core.PendingBlockVariant {
+		return pending, nil
+	} else {
+		// If pre_confirmed, network is polling pre_confirmed block and running on >= 0.14.0
+		latestHeader, err := h.bcReader.HeadsHeader()
+		if err != nil {
+			return nil, err
+		}
+
+		return emptyPendingForParent(latestHeader).AsPendingData(), nil
```

Comment:
I am not very sure what we are returning here. Maybe I am missing something, when asking for pending in 0.14.0 shouldn't we return `latest`?